FileSystem/Util.js

```diff
@@ -38,3 +38,30 @@ function deleteFile(name) {
     DriveApp.removeFile(fileList.next());
   }
 }
+
+/**
+ * Computes the Drive path for a file/folder using their Id
+ * @param {String} id The drive destination Id whose path needs to be computed
+ * @return {String} drivePath The corresponding drive path
+ */
+function getFullDrivePath(id, isFile) {
```

Comment:
This function is borrowed from @muskangarg98

Reply:
Update: We can have a new library which has these APIs and we set as a dependency in the FileMapper as well as FileSystem.

Reply:
Yeah, we can do that.

Reply:
@muskangarg98 I have raised a new issue on this, check #29